.github/workflows/linux.yml

```diff
@@ -24,18 +25,14 @@ jobs:
             echo "invalid GO version"
             exit 1
           fi
-          echo "::set-output name=GO_VERSION::$go_version"
-      - uses: actions/setup-go@v2
+          echo "GO_VERSION=$go_version" >> $GITHUB_OUTPUT
+      - uses: actions/setup-go@v3
         with:
           go-version: ${{ steps.get-go-version.outputs.GO_VERSION }}
-      - uses: actions/checkout@v2
-        with:
-          submodules: true
```

Comment:
Curious: Was this used to check out ecs cni and vpc cni submodules previously? Do we not need this anymore?